engine/class_modules/priest/sc_priest.cpp

```diff
@@ -523,20 +528,13 @@ struct smite_t final : public priest_spell_t
 
   smite_t( priest_t& p, util::string_view options_str )
     : priest_spell_t( "smite", p, p.find_class_spell( "Smite" ) ),
-      holy_fire_rank2( priest().find_rank_spell( "Holy Fire", "Rank 2" ) ),
       holy_word_chastise( priest().find_specialization_spell( 88625 ) ),
-      smite_rank2( priest().find_rank_spell( "Smite", "Rank 2" ) ),
```

Comment:
remove these from the top of the struct as well please, lines 521 and 523

Reply:
done